internal/host/hostcommands/install_cmd.go

```diff
@@ -331,15 +334,16 @@ func constructHostInstallerArgs(cluster *common.Cluster, host *models.Host, inve
 		// rd.neednet=1 console=ttysclp0 coreos.live.rootfs_url=http://172.23.236.156:8080/assisted-installer/rootfs.img zfcp.allow_lun_scan=0
 		// rd.znet=qeth,0.0.bdd0,0.0.bdd1,0.0.bdd2,layer2=1 rd.zfcp=0.0.8004,0x500507630400d1e3,0x4000404800000000
 		// random.trust_cpu=on rd.luks.options=discard ignition.firstboot ignition.platform.id=metal coreos.inst.persistent-kargs="console=tty1 console=ttyS1,115200n8"
+		log.Debugf("Check current boot cmdline: %s\n", strings.TrimSpace(inventory.Boot.CommandLine))
 		for _, part := range strings.Split(strings.TrimSpace(inventory.Boot.CommandLine), " ") {
 			if strings.HasPrefix(strings.ToLower(part), ZVM_NEEDNET) ||
-				strings.HasPrefix(strings.ToLower(part), ZVM_IP) ||
-				strings.HasPrefix(strings.ToLower(part), ZVM_NAMESERVER) ||
 				strings.HasPrefix(strings.ToLower(part), ZVM_LUN_SCAN) ||
 				strings.HasPrefix(strings.ToLower(part), ZVM_ZNET) ||
 				strings.HasPrefix(strings.ToLower(part), ZVM_DASD) ||
 				strings.HasPrefix(strings.ToLower(part), ZVM_FCP) {
 				installerArgs = append(installerArgs, "--append-karg", part)
+			} else if !hasStaticNetwork && (strings.HasPrefix(strings.ToLower(part), ZVM_IP) || strings.HasPrefix(strings.ToLower(part), ZVM_NAMESERVER)) {
```

Comment:
Errors in this place are not seen by the user - only if he goes to agent logs.
In addition it is an error while the cluster is installing which we would like to avoid.
If there is such a problem that can fail installation - it should be a pre-install validation.

Reply:
At the moment there is no additional pre-install validation for zVM  or LPAR nor KVM(specific to s390x). Do you want me to introduce it (at the moment parsing the cmdline is done only here)?

Reply:
We can either prevent the user to create such cluster, or validate it using pre-install validation.  Anyway, this is not the place to check it.

Reply:
So we need to add this to host_validation like CPU, memory, etc. to check s390x specific validations. Correct?

Reply:
As discussed remove IP and NAMESERVER. Will be documented.